fitbenchmarking/fitbenchmarking
# Default minimizer docs

#### Description of Work

Fixes #863 


#### Testing Instructions

1. Check that additions to docs make sense
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
